env.example.dev

```diff
@@ -26,3 +26,11 @@ ALLOW_ADMIN_CHANGES=true
 DISALLOW_ROBOTS=true
 
 PRIMARY_SITE_URL=http://gigadb.host
+
+OBJECT_STORAGE_ENDPOINT="https://s3.ap-northeast-1.wasabisys.com/"
+OBJECT_STORAGE_BUCKET="cms-assets"
+OBJECT_STORAGE_KEY="xxx"
+OBJECT_STORAGE_SECRET_KEY="xxx"
+OBJECT_STORAGE_REGION="xxx"
+OBJECT_STORAGE_REGION="ap-northeast-1"
```

Comment:
Perhaps only one OBJECT_STORAGE_REGION variable is required here.

Reply:
Yes, correct; I must have miscopied something. I can fix this.